### DISCUSSION TITLE
Live-server not able to update page while using Tailwind CLI with --watch option

### DISCUSSION THREAD

**Discussion by mnaderian** [tailwindlabs/tailwindcss#5168] [Help]
### What version of Tailwind CSS are you using?

v2.2.7

### What build tool (or framework if it abstracts the build tool) are you using?

Tailwind CLI

### What version of Node.js are you using?

v12.18.2 and npm v7.20.3

### What browser are you using?

Chrome, Firefox

### What operating system are you using?

Ubuntu Linux 20.10

### Reproduction repository

https://github.com/mnaderian/tailwind-signin.git

### Describe your issue

When using `live-server` and `npx tailwind -o res/styles.css --watch` command at the same time, `live-server` acts funny and not able to refresh the page and you have to refresh it manually!

**Comment by adamwathan** [comment]
Hey! Can reproduce but for me it looks like it's the HTML that's not being updated — I change a class and save and the old class name is still there in the HTML. I'm not sure we're going to be able do anything on our side to resolve this as it seems like a live-server issue — they are the ones responsible for refreshing the page, we just compile the CSS and the CSS is compiling correctly.

I'm going to convert to a discussion in case someone else has any thoughts on how to work around it but unfortunately I don't think it makes sense to keep this open as an issue as the problem is outside the scope of what Tailwind is actually doing.
